### PR TITLE
C1 abdelrahman add post

### PR DESCRIPTION
**Pull request by abdelrahmanMaged**
@BeshoSmile  would you please review the code for the two view view_channel and view_subchannel?? 

Regards,

### CODE REVIEW

File: Profiles.txt [hunk 1]
```diff
@@ -85,4 +84,4 @@ Name: Heba Ahmed El Sobky
 Role: Developer/Reviewer
 Major: BI
 ID: 22-2379
-Interests: Knitting and playing piano
```
Comment (BeshoyAtef) [inline Profiles.txt:88]: hwa eh dah ?

**Comment by BeshoyAtef** [comment]
Abdo your code looks fine bs why you added the all project ? why not views only ?

**Comment by abdelrahmanMaged** [comment]
sorry i was testing git however yesterday i added the specific files and
folder for templates and static files

**Comment by BeshoyAtef** [comment]
please next time only only commit the needed files

**Comment by BeshoyAtef** [comment]
#96 Code reviewed

**Pull request merged**